src/Quotebot.Data/CosmosDbService.cs

```diff
@@ -19,7 +20,7 @@ public async Task<bool> TryCreateQuoteRecord(Quoted message)
         {
             try
             {
-                var result = await _container.CreateItemAsync(message, new PartitionKey(message.Author.Id));
+                var result = await _container.CreateItemAsync(message, new PartitionKey(message.Author.Id)).ConfigureAwait(false);
```

Comment:
I think you can drop the ConfigureAwait(false). It's only needed if this will be used by older code, or if you're using SynchronizationContext (haven't looked at other files to see if that's the case yet)

Reply:
ConfigureAwait(false) will allow the `GetAwaiter` to return on a different thread instead of holding a thread id indefinitely.